core/services/assets/I18nRegistry.php

```diff
@@ -2,15 +2,14 @@
 
 namespace EventEspresso\core\services\assets;
 
+use EventEspresso\core\domain\Domain;
 use EventEspresso\core\domain\DomainInterface;
 
 /**
- * I18nRegistry
- * Handles registering translations for registered script handles.
- *
  * @package EventEspresso\core\services\assets
  * @author  Darren Ethier
  * @since   4.9.62.p
+ * @deprecated $VID:$
  */
 class I18nRegistry
```

Comment:
totally nerfed this entire class 😆 because none of it is needed now

Reply:
Cool 🚀